packages/dev/eslint-config-browser/package.json

```diff
@@ -7,6 +7,6 @@
   },
   "peerDependencies": {
     "@babel/core": "^7.0.0",
-    "eslint": ">= 7.0.0"
+    "eslint": "^7.32.0"
```

Comment:
Had to ensure `eslint` was the same version in all packages that use it due to it being a global workspace package.